server/src/__tests__/completions.test.ts

```diff
@@ -1259,4 +1259,60 @@ describe('On Completion', () => {
       ])
     )
   })
+
+  it('Provide proper completion items on LICENSE variable', async () => {
+    analyzer.analyze({
+      uri: DUMMY_URI,
+      document: FIXTURE_DOCUMENT.COMPLETION
+    })
+
+    const resultBeforeResolve = onCompletionHandler({
+      textDocument: {
+        uri: DUMMY_URI
+      },
+      position: {
+        line: 31,
+        character: 17
+      }
+    })
+
+    const licenseTested = 'GPL-2.0-with-bison-exception'
+
+    const expectedResultBeforeResolve = {
+      label: licenseTested,
+      kind: 12,
+      deprecated: true,
+      labelDetails: {
+        description: 'Source: SPDX License List'
+      },
+      documentation: 'Loading...',
+      textEdit: {
+        range: {
+          start: { line: 31, character: 11 },
+          end: { line: 31, character: 28 }
+        },
+        newText: licenseTested
+      }
+    }
+
+    expect(resultBeforeResolve).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining(expectedResultBeforeResolve)
+      ])
+    )
+
+    const completionItem = resultBeforeResolve.find((item) => item.label === licenseTested)
+
+    if (completionItem === undefined) {
+      throw new Error('completionItem is undefined')
+    }
+
+    const resultAfterResolve = await onCompletionResolveHandler(completionItem)
+    expect(resultAfterResolve).toEqual(
+      expect.objectContaining({
+        ...expectedResultBeforeResolve,
+        documentation: "Bison Exception\n\nAs a special exception, you may create a larger work that contains part or all of the Bison parser skeleton and distribute that work under terms of your choice, so long as that work isn't itself a parser generator using the skeleton or a modified version thereof as a parser skeleton. Alternatively, if you modify or redistribute the parser skeleton itself, you may (at your option) remove this special exception, which will cause the skeleton and the resulting Bison output files to be licensed under the GNU General Public License without this special exception.\n\nThis special exception was added by the Free Software Foundation in version 2.2 of Bison.\n\n"
```

Comment:
you can use expect.stringContaining to make this a bit more concize

Reply:
done